internal/action/config.go

```diff
@@ -30,45 +32,31 @@ func (s *Action) Config(c *cli.Context) error {
 		return exit.Error(exit.Usage, nil, "Usage: %s config key value", s.Name)
 	}
 
-	if err := s.setConfigValue(ctx, c.Args().Get(0), c.Args().Get(1)); err != nil {
+	if err := s.setConfigValue(ctx, store, c.Args().Get(0), c.Args().Get(1)); err != nil {
 		return exit.Error(exit.Unknown, err, "Error setting config value")
 	}
 
 	return nil
 }
 
-func (s *Action) printConfigValues(ctx context.Context, needles ...string) {
-	m := s.cfg.ConfigMap()
-	for _, k := range filterMap(m, needles) {
+func (s *Action) printConfigValues(ctx context.Context, store string, needles ...string) {
+	// debug.Log("printing value of %v", needles)
```

Comment:
Why leave this commented? I'd say either leave it as a debug log or remove it, no?
Overall I'd leave it as a debug log, feels it could be useful in the coming months.